PowerPointLabs/PowerPointLabs/TimerLab/TimerLabPaneWPF.xaml.cs

```diff
@@ -430,68 +404,23 @@ private void WidthSlider_Loaded(object sender, RoutedEventArgs e)
             WidthSlider.Value = TimerLabConstants.DefaultTimerWidth;   
         }
 
-        private void WidthSlider_PreviewMouseDown(object sender, System.Windows.Input.MouseButtonEventArgs e)
-        {
-            if (HasTimer())
-            {
-                ReformMissingComponents();
```

Comment:
ReformMissingComponents was changed to only 5 checks for null, so it should not be resource heavy. On the other hand, it seems like setting the Left, Top, and Width too frequently is causing the performance issues, I will work on that.
![123](https://cloud.githubusercontent.com/assets/19281538/26137090/975197d2-3af2-11e7-8d9c-97e7304f46a6.PNG)